site/earlgrey-diagram/src/metrics.js

```diff
@@ -0,0 +1,711 @@
+export default {
+    "adc_ctrl": {
+      "design_spec": "../doc",
+      "dv_doc": "../doc/dv",
+      "hw_checklist": "../doc/checklist",
+      "name": "adc_ctrl",
+      "revisions": [
+        {
+          "commit_id": null,
+          "design_stage": "D3",
+          "dif_stage": "S2",
+          "life_stage": "L1",
+          "verification_stage": "V2S",
+          "version": "1.0"
+        }
+      ],
+      "sw_checklist": "/sw/device/lib/dif/dif_adc_ctrl"
+    },
```

Comment:
All these entries get auto-populated, right? If so, I suggest adding a comment at the beginning of each document or section that is generated

Reply:
@jprendes Could you add a comment to the top of this file as a suggested change? And I will add it to the PR. Thanks